build/move-release-notes

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+const authenticate = require('./github-authentication')
+const { program } = require('commander')
+
+const RELEASE_NOTES_DIR = 'release-notes'
+const UNRELEASED_DIR = 'unreleased'
+
+program
+  .option('-d, --debug', 'enable extreme logging')
+  .requiredOption('--deliverino-private-key <deliverino-private-key>', 'private key of the deliverino github app in pem format with base64 encoding')
+  .requiredOption('--owner <owner>', 'owner of the current repository, usually "Integreat"')
+  .requiredOption('--repo <repo>', 'the current repository, usually "integreat-webapp" or "integreat-react-native-app"')
+  .requiredOption('--branch <branch>', 'the current branch')
+  .requiredOption('--commit-sha <commit-sha>', 'the sha1 of the current commit')
+
+const moveReleaseNotes = async ({ newVersionName, deliverinoPrivateKey, owner, repo, branch, commitSha }) => {
+  const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
+  const lastCommit = await appOctokit.git.getCommit({ owner, repo, commit_sha: commitSha })
+
+  // Tree of the root folder of the project
+  const rootTreeSha = lastCommit.data.tree.sha
+  const rootTree = await appOctokit.git.getTree({ owner, repo, tree_sha: rootTreeSha })
+
+  // Tree of the 'release-notes' folder of the project
+  const releaseNoteTreeSha = rootTree.data.tree.find(it => it.path === RELEASE_NOTES_DIR).sha
```

Comment:
it could occur, that the release notes are not found, becouse getTree truncates if there are too many elements(?). Could you add a check, that if the release notes directory cannot be found we check whether `rootTree.data.truncated` is true and throw an Error message.
Even though I don't think the chances are high of thet occuring I still think it is better to be prepared for stupid bugs like that :)

Reply:
`If truncated is true in the response then the number of items in the tree array exceeded our maximum limit. If you need to fetch more items, use the non-recursive method of fetching trees, and fetch one sub-tree at a time.` Since we don't use recursive fetch, I think we should be fine here. But I'll check and throw an error.

Reply:
Nice 👍